extensions/integration-tests/src/test/utils.ts

```diff
@@ -180,6 +180,27 @@ export async function runQuery(query: string, ownerUri: string): Promise<azdata.
 
 }
 
+export async function retryFunction(fn: () => Promise<any>, retryCount: number): Promise<any> {
+	let result: any;
+	while (retryCount > 0) {
+		--retryCount;
+
+		try {
+			result = await fn();
+			if (result) {
```

Comment:
Why are you checking for truthiness here? What if I have a void function? I think as long as the function executes without throwing it should return

`return await fn();`